src/main/java/com/linkedin/kmf/services/ConsumeMetrics.java

```diff
@@ -64,7 +64,7 @@
       Sensor topicPartitionCount = metrics.sensor("topic-partitions");
       topicPartitionCount.add(
           new MetricName("topic-partitions-count", METRIC_GROUP_NAME, "The total number of partitions for the topic.", tags),
-          new Total(partitionCount));
+          new Total());
```

Comment:
Is this necessary for the version bump?

Reply:
yep!

Reply:
the new version of kafka-clients makes the old `new Total(partitionCount))` usage deprecated

Reply:
Interesting

Reply:
I checked again. It seems like since 2.4, the replacement for Total is a CumulativeSum. updated. `@deprecated since 2.4 . Use {@link CumulativeSum} instead.`